addons/block_code/ui/script_window/script_window.tscn

```diff
@@ -0,0 +1,63 @@
+[gd_scene load_steps=3 format=3 uid="uid://v3nopbe4l0ph"]
+
+[ext_resource type="Script" path="res://addons/block_code/ui/script_window/script_window.gd" id="1_jja22"]
+
+[sub_resource type="LabelSettings" id="LabelSettings_lhqvi"]
+
+[node name="ScriptWindow" type="Window"]
+initial_position = 1
+size = Vector2i(750, 750)
+transient = true
+script = ExtResource("1_jja22")
+
+[node name="Margin" type="MarginContainer" parent="."]
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+theme_override_constants/margin_left = 25
+theme_override_constants/margin_top = 25
+theme_override_constants/margin_right = 25
+theme_override_constants/margin_bottom = 25
+
+[node name="VBox" type="VBoxContainer" parent="Margin"]
+layout_mode = 2
+
+[node name="Label" type="Label" parent="Margin/VBox"]
+layout_mode = 2
+size_flags_horizontal = 0
+size_flags_vertical = 0
+text = "Here is what your Block Code looks like in GDScript!"
```

Comment:
I think this would be better as the window title instead of a label in the window. I think I'd probably make it something a little more generic like `Block Code Generated GDScript`. But we can always fine tune it after merging.